developer_guide/installation/partners_portal_archive.rst

```diff
@@ -0,0 +1,42 @@
+Obtain Akeneo PIM Enterprise Edition Archive
+============================================
+
+This document provides step by step instructions to obtain the PIM Enterprise Edition with the archive.
+If you already have your archive please follow the installation guidelines: :ref:`installation_archive <extracting-the-archive>`.
+
+Getting Akeneo Partners Portal access
+-------------------------------------
+After obtaining your Enterprise Edition license you should have received an access to the Akeneo Partners Portal.
+
+If you haven't received it you could request it at: http://helpdesk.akeneo.com/
+
+If you are a developer on the project you should request an account to access to the project to your company manager (supervisor).
+
+Setting Enterprise Edition access on the Partners Portal
+--------------------------------------------------------
+By default composer uses packagist.org to retrieve all open source packages and their updates.
```

Comment:
Composer (capital letter + you should add a link to https://getcomposer.org/ on the word)